ui/components/multichain/pages/page/index.scss

```diff
@@ -1 +1,17 @@
 @import 'components/';
+
+.multichain-page {
+  &__inner-container {
+    max-width: 480px;
+  }
+
+  &-header {
+    top: 0;
+    position: sticky;
+  }
+
+  &-footer {
+    bottom: 0;
+    position: sticky;
+  }
```

Comment:
I think these styles may be unnecessary and could be removed? The position: sticky; property is used to make an element stick to the top or bottom of its parent container or the viewport when you scroll. However, if the scrolling is confined to the <Content /> component, the <Header /> and <Footer /> components will always remain in their original positions, regardless of the scrolling within the <Content /> component.

Therefore, in this specific scenario, using position: sticky; on the <Header /> and <Footer /> components would not have any effect.
```suggestion
```